documentation/src/docs/controller-service.md

```diff
@@ -517,275 +494,253 @@ the lock on the resource and hence permission to process the task. The
 parameters for executing a task are serialized and stored under the
 resource.
 
-Currently we use task framework only for create stream tasks. All the
-other background processing is done using event processor framework.
+Currently we use Task framework only to create Stream tasks. All the
+other background processing is done using Event Processor framework.
 
 ### Event Processor Framework
 
-Event processors framework is a background worker sub system which reads
-events from an internal stream and processes it, hence the name event
-processor. All event processors in our system provide **at least once
+Event processors framework is a background worker subsystem which reads
```

Comment:
`The Event Processors Framework...` I'm trying to follow the capitalization that @Aparnarr is using.

Reply:
`In Pravega, all Event Processors provide...`.

Reply:
@shiveshr should we argue that it is idempotent, so multiple executions do not matter?

Reply:
The event processor itself is not idempotent.. the workflows implemented on the event processor are idempotent.

Reply:
@shiveshr Any changes required here??

Reply:
@shiveshr Edited!.Please check